Microsoft.HealthVault.Fhir/Transformers/PractitionerToHealthVault.cs

```diff
@@ -44,6 +44,12 @@ private static ItemTypes.PersonItem ToPerson(this Hl7.Fhir.Model.Practitioner fh
             {
                 foreach (var address in fhirPractitioner.Address)
                 {
+                    if ((string.IsNullOrEmpty(address.City))||(string.IsNullOrWhiteSpace(address.Line.ToString()))||(string.IsNullOrWhiteSpace(address.PostalCode))
```

Comment:
If this is a requirement for converting the address to healthvault, this check should be completed in the tohealthvault() method instead of before the call.